pyiceberg/io/pyarrow.py

```diff
@@ -154,10 +154,9 @@
 ONE_MEGABYTE = 1024 * 1024
 BUFFER_SIZE = "buffer-size"
 ICEBERG_SCHEMA = b"iceberg.schema"
-FIELD_ID = "field_id"
-DOC = "doc"
-PYARROW_FIELD_ID_KEYS = [b"PARQUET:field_id", b"field_id"]
-PYARROW_FIELD_DOC_KEYS = [b"PARQUET:field_doc", b"field_doc", b"doc"]
+# The PARQUET: in front means that it is Parquet specific, in this case the field_id
+PYARROW_FIELD_ID_KEY = b"PARQUET:field_id"
```

Comment:
Shall we name it as `PYARROW_PARQUET_FIELD_ID_KEY` since it is parquet specific? If we add ORC Fileformat support later(https://github.com/apache/iceberg-python/issues/20), we may want a `PYARROW_ORC_FIELD_ID_KEY = b"iceberg.id"` as the physical schema looks like
```
user_id: int32
  -- field metadata --
  iceberg.id: '1'
  iceberg.required: 'false'
user_name: string
  -- field metadata --
  iceberg.id: '2'
  iceberg.required: 'false'
status: string
  -- field metadata --
  iceberg.id: '3'
  iceberg.required: 'false'
cost: double
  -- field metadata --
  iceberg.id: '4'
...
```

Reply:
Thanks, that's a good point! I've updated the code 👍